testsuite/docs-examples-cpp/src/docs-examples-imagebufalgo.cpp

```diff
@@ -267,6 +267,23 @@ void example_circular_shift()
 // Section: Import / export
 
 
+void example_make_texture()
+{
+    // BEGIN-imagebufalgo-make-texture
+    ImageBuf Input ("texture_input.exr");
```

Comment:
Just change this to grid.exr, and you won't need the extra copy.

It's ok to alter the examples if you think there's a way to clarify or simplify them.